src/tests/ftest/util/mpio_utils.py

```diff
@@ -126,6 +126,20 @@ def run_llnl_mpi4py_hdf5(self, hostfile, pool_uuid, test_repo,
             cmd = " ".join(test_cmd)
         elif test_name == "mpi4py" and \
              os.path.isfile(os.path.join(test_repo, "test_io_daos.py")):
+            cmd = "daos cont create --pool={} --svc={} --type=POSIX".format(
```

Comment:
With this test is it always expected that the pool service replica rank for the pool with uuid pool_uuid will be rank 0?

Reply:
since it's run on the small cluster with 1 server, yes.
however i agree that all these tests should use whatever is returned from pool create. It's not related to this patch because this value is hardcoded to 0 in many places.